vms/platformvm/blocks/executor/acceptor.go

```diff
@@ -34,115 +34,43 @@ type acceptor struct {
 }
 
 func (a *acceptor) BanffAbortBlock(b *blocks.BanffAbortBlock) error {
-	a.ctx.Log.Debug(
-		"accepting block",
-		zap.String("blockType", "banff abort"),
-		zap.Stringer("blkID", b.ID()),
-		zap.Uint64("height", b.Height()),
-		zap.Stringer("parentID", b.Parent()),
-	)
```

Comment:
I feel like reducing all these super similar logs was in-and-of itself a good change.